.rubocop.yml

```diff
@@ -57,3 +57,12 @@ RSpec/AnyInstance:
   Enabled: false
 RSpec/FactoryBot/SyntaxMethods:
   Enabled: false
+# RSpec/ContextWording:
```

Comment:
Please uncommend this parameter, all contexts should starts with these prefixes

Reply:
Uncommented parameter ContextWording